app/graphql/queries/survey_query.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Queries
+  class SurveyQuery < BaseQuery
+    description 'Survey list'
+
+    type [Types::SurveyType], null: false
+
+    def resolve
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/080830d571ead7083d5b15ab6138b613fcba3462/app/graphql/queries/survey_query.rb#L9"></span>Doesn’t depend on instance state (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>